libdnf/dnf-repo.cpp

```diff
@@ -1473,9 +1473,10 @@ dnf_repo_check_internal(DnfRepo *repo,
 
     /* init newRepo */
     auto newRepo = hy_repo_create(priv->repo->getId().c_str());
+    auto newRepoImpl = libdnf::repoGetImpl(newRepo);
+    newRepoImpl->additionalMetadata = repoImpl->additionalMetadata;
```

Comment:
Please could you add a note why you copy it? And why you do not copy other elements?